### ISSUE TITLE
Portal 2 Bug: prop_laser_catcher does not detect lasers correctly when angled

### ISSUE THREAD

**Issue by LambdaCore21** [StrataSource/Portal-2-Community-Edition#1306]
### Describe the bug

Portal 2 Bug: prop_laser_catcher does not detect lasers correctly when angled

![picang](https://user-images.githubusercontent.com/63379443/235443557-0b0b6a2a-7e81-4569-b538-71045cc0ff14.png)
[angled_catcher map bsp.zip](https://github.com/StrataSource/Portal-2-Community-Edition/files/11365018/angled_catcher.map.bsp.zip)

### To Reproduce

1. Open the attached map

3. Play around with the cube

**Comment by FrozenDragon0** [comment]
Seems like the actual hitbox the laser needs to touch isn't being angled correctly despite the catcher model being rotated, though that's just my speculation.

**Comment by Luke18033** [comment]
That is probably the issue, the laser target entity uses an AABB to detect lasers - what does it look like if you do `developer 1;ent_bbox point_laser_target`?

**Comment by LambdaCore21** [comment]
![grafik](https://user-images.githubusercontent.com/63379443/235613216-02b0ecde-0a93-4519-818b-ad8298b01a2a.png)

**Comment by Luke18033** [comment]
Yep, the catcher will only activate when the beam is touching that box, which matches up with what your original screenshots showed. It should probably be changed to use an oriented bounding box so this works correctly.

**Comment by LambdaCore21** [comment]
Related to hitbox isssue #1541
